diff --git a/pkg/imager/profile/default.go b/pkg/imager/profile/default.go
index 5c7b6d645..6bc5fd6b6 100644
--- a/pkg/imager/profile/default.go
+++ b/pkg/imager/profile/default.go
@@ -14,7 +14,7 @@ const (
 	mib = 1024 * 1024
 
 	// MinRAWDiskSize is the minimum size disk we can create. Used for metal images.
-	MinRAWDiskSize = 1246 * mib
+	MinRAWDiskSize = 1646 * mib
 
 	// DefaultRAWDiskSize is the value we use for any non-metal images by default.
 	DefaultRAWDiskSize = 8192 * mib
diff --git a/pkg/machinery/imager/quirks/partitions.go b/pkg/machinery/imager/quirks/partitions.go
index 4247d8016..2a1ad8357 100644
--- a/pkg/machinery/imager/quirks/partitions.go
+++ b/pkg/machinery/imager/quirks/partitions.go
@@ -16,7 +16,7 @@ const (
 
 // GrubEFISize return EFI partition size for GRUB layout.
 func (p PartitionSizes) GrubEFISize() uint64 {
-	return 100 * mib
+	return 500 * mib
 }
 
 // GrubBIOSSize return BIOS GRUB partition size.
